Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM ubuntu:17.04
```

Comment:
The ninja version is too old in TLS, NNPACK can't compile. Docker CE stable version only support 1 year, so it's meaningless for using ubuntu LTS.